nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExtractGrok.java

```diff
@@ -102,6 +103,15 @@
         .addValidator(StandardValidators.FILE_EXISTS_VALIDATOR)
         .build();
 
+    public static final PropertyDescriptor KEEP_EMPTY_CAPTURES = new PropertyDescriptor.Builder()
+            .name("Keep Empty Captures")
+            .description("If true, then empty capture values will be included in the returned capture map.")
+            .required(false)
```

Comment:
since it's an "allowable values"-type of property, we usually set required=true

Reply:
done